src/elements/PriceRange/PriceRange.tsx

```diff
@@ -29,7 +29,7 @@ export class PriceRange extends React.Component<PriceRangeProps> {
     return (
       <LabeledRange
         formatter={this.formatter.bind(this)}
-        label="Price"
+        label={this.props.currency}
```

Comment:
I haven't seen this change in any design comps. Are we changing the label for this component from "Price" to "USD" or "GBP"?

Reply:
Yeah! It's part of the change in which price becomes a drop-down (it wasn't before). Shows up in this comp: [Zeplin](https://zpl.io/aByJzZe)